adamacs/behavior.py

```diff
@@ -113,8 +93,50 @@ class TrialVariable(dj.Part):
 		variable_value: varchar(2000)
 		"""
 
+@schema
+class BehaviorIngest(dj.Imported):
+	definition = """
+	-> session.Recording
+	"""
+  @property
+  def key_source(self): #opional means of subsetting or reordering keys worth ingesting
+  	good_records = session.Recording.fetch('KEY') #& session.Qualitycontrol
+  	return good_records
+
```

Comment:
```suggestion
```

Let's remove it here, modifying `key_source` is mostly not needed. We can add when needed